### ISSUE TITLE
What is the difference between MultKAN.py and KAN.py and which file should I start with to understand the code?

### ISSUE THREAD

**Issue by YYD1319** [KindXiaoming/pykan#344]

**Comment by MeDott29** [comment]
How's your code understanding journey going?

**Comment by YYD1319** [comment]
> How's your code understanding journey going?

I see the code in line 1671, I don't understand what is the purpose of this. KAN is used for instantiation in hellokan.ipynb, but the actual execution is done by MultKAN, so I choose to understand MultKAN.

**Comment by KindXiaoming** [comment]
Yes, KAN and MultKAN are the same thing. The point is that some users may already get used to the name `KAN`, and we want to make KAN a general framework rather than the original sum-only networks.

**Comment by YYD1319** [comment]
> Yes, KAN and MultKAN are the same thing. The point is that some users may already get used to the name `KAN`, and we want to make KAN a general framework rather than the original sum-only networks.

I see it, thanks!
